sites/antwerpen/macros.tex

```diff
@@ -12,7 +12,7 @@
 \macro{\loginnode}{login.hpc.uantwerpen.be}
 \macro{\loginhost}{ln02.hopper.uantwerpen.vsc}
 %TODO: RSA key fingerprint here
-\macro{\loginfingerprint}{20:96:ca:e7:f0:62:87:34:ca:9c:71:84:b7:d3:46:f8}
```

Comment:
Did you check whether `loginfingerprint` is still used somewhere?

Reply:
Yes, grepped through the source